Sources/armory/renderpath/Inc.hx

```diff
@@ -531,12 +545,20 @@ class Inc {
 	}
 
 	public static inline function getVoxelResZ(): Float {
-		#if (rp_voxelgi_resolution_z == 1.0)
```

Comment:
If we need this many, maybe just output the float value for `rp_voxelgi_resolution_z` directly alongside these lines? https://github.com/armory3d/armory/blob/main/blender/arm/write_data.py#L468

Reply:
> There still could be a change of resolution while playing

Can you elaborate on that? It looks like this function just converts the compile-time `rp_voxelgi_resolution_z` define into float.

Reply:
I was thinking of the live patch mode.
Res is written to compiled.inc, read from renderpath and converted to floats.
writing float directly into the file will not change the need for a function in Inc.

Reply:
The `#if`s are resolved at compile time, nothing to do with performance. I was asking to replace multiple source code lines with a single one. 🙂

Reply:
It's strange that we can test rp_voxelgi_resolution against an integer via a macro, but can't return it's value straight away.

This would go like:```

public static inline function getVoxelRes(): Int {
	return rp_voxelgi_resolution;
}
```

`Unknown identifier : rp_voxelgi_resolution`

Reply:
I will look at that at the end of the afternoon

Reply:
I'm not able to get preprocessor expressions values.
This change is irrelevant though, if you need to fix this do so.